api/src/main/java/org/pmiops/workbench/db/model/DbUserRecentlyModifiedResource.java

```diff
@@ -27,6 +28,10 @@
   private DbUserRecentlyModifiedResourceType resourceTypeEnum;
   private String resourceId;
   private Timestamp lastAccessDate;
+  private DbCohort cohort;
```

Comment:
what about cohort review?

Reply:
these fields don't represent actual DB relationships, so I think it will be confusing to include them here.

what do you think about applying this relationship using something like populateRecentlyModifiedResources() at mapping time?

Reply:
I started with the hope that i can join them at DAO hence kept it there. 
I might check with Brian as well once. If that doesnt work, your suggestion is to just have the resource_type/id in dao (a true representation of the table). At the time of mapping it to the Modal object, get the appropriate  Resource Objects and convert them?

Reply:
Yes exactly.  InstitutionService does something like this.